### ISSUE TITLE
Does this caravan area have a sanitary_dump_station ?

### ISSUE THREAD

**Issue by petitminion** [streetcomplete/StreetComplete#5233]
This is a proposition to tag objects tagged with this tag : [caravan_site](https://wiki.openstreetmap.org/wiki/Tag:tourism%3Dcaravan_site)

### General
Affected tag(s) to be modified/added:   [sanitary_dump_station](https://wiki.openstreetmap.org/wiki/Key:sanitary_dump_station)
Question asked: **Does this caravan area have a  [sanitary_dump_station](https://wiki.openstreetmap.org/wiki/Key:sanitary_dump_station)?**
response : yes/no/custumors 

same could be done for theses tags  :
- [water_point](https://wiki.openstreetmap.org/wiki/Key:water_point)
- [power_supply](https://wiki.openstreetmap.org/wiki/Key:power_supply)

### Checklist
Checklist for quest suggestions (see [guidelines](https://github.com/streetcomplete/StreetComplete/blob/master/QUEST_GUIDELINES.md)):
- [ x ] 🚧 To be added tag is established and has a useful purpose
- [ x ] 🤔 Any answer the user can give must have an equivalent tagging (Quest should not reappear to other users when solved by one)
- [ x ] 🐿️ Easily answerable by any pedestrian from the outside but a survey is necessary
- [ x ] 💤 Not an overwhelming percentage of quests have the same answer (No spam)
- [ x ] 🕓 Applies to a reasonable number of map data (Worth the effort)

### Ideas for implementation

**Element selection:** 

**Metadata needed:** 

**Proposed UI:**

**Comment by mnalis** [comment]
- (ref: https://github.com/streetcomplete/StreetComplete/issues/5206)

**Comment by mnalis** [comment]
@petitminion Although, it would probably be a better idea to open a separate issue for each of the quests requested (IOW, keep it one-quest-per-issue).  i.e. keep this one exclusively about [sanitary_dump_station](https://wiki.openstreetmap.org/wiki/Key:sanitary_dump_station), and create two new issues for other two quests.

Because implementing one of them would likely close the whole issue (as seen #5206), and even if doesn't, it will make discussing several different quests in the same issue hard to follow.

**Comment by rhhsm** [comment]
I don't think it fulfils "Easily answerable by any pedestrian from the outside but a survey is necessary": I wouldn't know how to look for one. Are they visible from the outside (outside the area, if it is not publicly accessible)?

**Comment by petitminion** [comment]
yes pedestrians can easily access a caravan_site :)
 
Also most of the time there is a sign indicating if the service is present : 
- french sign : https://www.techni-contact.com/ressources/images/produits/zoom/panneau-station-de-vidange-pour-camping-car-ce24-12231473-1.jpg 
- US, english ? https://thumbs.dreamstime.com/b/rv-camper-motor-home-sewage-dump-station-netherlands-177670508.jpg

Sadly I didn't find any open source image to add to the wiki but I will do it when  I find a panel on the road o/

**Comment by qugebert** [comment]
> If I understand correctly linked issue (and judging from the picture at [tourism=caravan_site](https://wiki.openstreetmap.org/wiki/Tag:tourism%3Dcaravan_site), the caravans are prevented by entering by having to pay for [barrier=lift_gate](https://wiki.openstreetmap.org/wiki/Tag:barrier%3Dlift_gate) or similar to allow the vehicle in, but pedestrians can roam around?

I only know a few of them, but yes, its mostly just some seperate parking space on a normal parking site with some additional equipment (dump station etc,). And we ask for various things on camping sites too, these are more likely not accessible for non-customers.

I can do the implementation if it is okay.

**Comment by qugebert** [comment]
So here is my draft.
I took the description from the wiki, is this ok or are there any suggestions for another text?
And are these two pictures enough? Isn't the quest form too high if we add another row?


![Screenshot_20231030-235233](https://github.com/streetcomplete/StreetComplete/assets/44033795/fd9eb4bd-6d5e-4eb0-9ca6-88b8bb5d8db7)

**Comment by rhhsm** [comment]
Looks good, including the quest pin. I'm glad you didn't use the word "caravan" because that would confuse me (and probably many others).
The sentence could be shortened a bit, maybe "Facility for depositing human waste from a toilet holding tank of a motorhome, long haul truck, etc. Might look like this:"

**Comment by qugebert** [comment]
@petitminion @mcliquid see https://github.com/streetcomplete/StreetComplete/pull/5363#issuecomment-1796571671
any ideas?

**Comment by qugebert** [comment]
> I can create a simple icon for you if you want to.

Sure, this would help a lot.
